index.html

```diff
@@ -22,350 +23,279 @@
-              <rect id="Rectangle 4" x="294" y="61.79" width="95.5804" height="22.021" transform="rotate(-30 294 61.79)" fill="#EBEBFF"/>
-            </g>
-          </svg>
-        </div>
-
-      <div id="header-background-mobile"></div>
-
-    <!-- Works section -->
-    <section id="works-section">
-
-      <div id="align-section">
+        <div id="div-social-media"><p class="social-media-p">LET'S CONNECT</p></div>
+
+        <ul id="social-media-icons-li">
+          <li id="social-media-icons-item-1">
+            <a href="https://twitter.com/AlejandroMaggi7"><img class="social" src="css/twitter.svg" alt="twitter" width="auto" height="auto"></a>
+          </li>
+          <li id="social-media-icons-item-2">
+            <a href="https://www.linkedin.com/in/alejandro-maggioni-086678b5/"><img class="social" src="css/linkedin.svg" alt="linkedin" width="auto" height="auto"></a>
+          </li>
+          <li id="social-media-icons-item-3">
+            <a href="#"><img class="social" src="css/m.svg" alt="m" width="auto" height="auto"></a>
+          </li>
+          <li id="social-media-icons-item-4">
+            <a href="https://github.com/alex1779/"><img class="social" src="css/github.svg" alt="github" width="auto" height="auto"></a>
+          </li>
+          <li id="social-media-icons-item-5">
+            <a href="#"><img class="social" src="css/hand.svg" alt="hand" width="auto" height="auto"></a>
+          </li>
+        </ul>
+      </div>
 
-        <div id="works-section-div1" class="works-section-class">
-          <div id="snapshot-portfolio-1" class="snapshot-portfolio-class"></div>
-          <div id="block-apart1">
-            <div id="block-apart-primary-text-1">
-              <div id="block-apart-primary-text-1-project-title">
-                <h1 id="block-apart-primary-text-1-project-title-text-1" class="block-apart-primary-text">Tonic</h1>
+      <!-- Works section -->
+      <section id="works-section">
+
+        <div id="align-section">
+
+          <div id="works-section-div1" class="works-section-class">
+            <div id="snapshot-portfolio-1" class="snapshot-portfolio-class"></div>
+            <div id="block-apart1">
+              <div id="block-apart-primary-text-1">
```

Comment:
Kindly modify the background of the works section to match your chosen design template. The works sections has a different background from the individual project cards.

Your design:
![image](https://user-images.githubusercontent.com/20554583/194477179-112aaf69-0455-4fa0-9d9b-6845b0ce64ba.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/194477079-a7b7f793-64e0-4122-8265-47f56f8e7855.png)